Automattic/wp-calypso
# Changing logo partner to Logotron

#### Changes proposed in this Pull Request

We're testing a new logo maker partnership with Logotron to see if our users enjoy that service more. Context available at pbBQWj-II-p2. The partnership is featured in the Quick Links section of My Home and on a card on Marketing Tools. This PR updates those areas.

The link is updated in D58673-code.

Screenshot of the My Home update
<img width="1077" alt="Screen Shot 2021-03-15 at 10 30 42 AM" src="https://user-images.githubusercontent.com/35781181/111170014-dec63200-8579-11eb-8097-9d56b975307d.png">

Screenshot of the updated card on Marketing Tools:
<img width="617" alt="Screen Shot 2021-03-15 at 10 33 46 AM" src="https://user-images.githubusercontent.com/35781181/111170129-fa313d00-8579-11eb-92b9-90f61ad281b8.png">

#### Testing instructions
* Patch your sandbox with D58673-code
* Update /etc/hosts to point wp.me to your sandbox
* Checkout this PR and start Calypso
* Navigate to My Home, confirm the quick link says `Logotron` and that the link goes to the Logotron+WPcom partnership page.
* navigate to `marketing/tools/SITESLUG` and confirm that the card now mentions Logotron and that the link goes to the partnership page.

Related to #
* D58673-code
